quickwit/quickwit-search/src/scroll_context.rs

```diff
@@ -222,6 +222,9 @@ impl FromStr for ScrollKeyAndStartOffset {
         let scroll_ulid = u128::from_le_bytes(scroll_ulid_bytes.try_into().unwrap()).into();
         let from = u64::from_le_bytes(from_bytes.try_into().unwrap());
         let max_hits = u32::from_le_bytes(max_hits_bytes.try_into().unwrap());
+        if max_hits > 10_000 {
```

Comment:
we can get there only if a user tried to forge a bad scroll ID right?

Reply:
yes